### ISSUE TITLE
[Fixed Position] Image telemetry should work properly in Fixed Position displays

### ISSUE THREAD

**Issue by charlesh88** [nasa/openmct#1740]
Example Imagery can be added to Fixed Position displays, but does not display properly when added.

It should show the url of the most recent image; instead it shows nothing and error messages are logged to console:

![screen shot 2017-10-02 at 10 50 42 am](https://user-images.githubusercontent.com/226503/31091442-da657230-a75f-11e7-8e93-ef627016db96.png)
![screen shot 2017-10-02 at 10 51 36 am](https://user-images.githubusercontent.com/226503/31091449-dd2cc1f8-a75f-11e7-8973-2e70fbba6cf8.png)

```
FixedController.js:364 Uncaught TypeError: Cannot read property 'source' of undefined
    at FixedController.updateView (FixedController.js:364)
    at FixedController.js:348
    at TelemetryAPI.js:273
    at Array.forEach (<anonymous>)
    at TelemetryAPI.js:272
    at plugin.js:64
```

**Comment by larkin** [comment]
@charlesh88 
**tldr;** I propose changing this issue to fix the bug and ensure the fixed position layout has reasonable behavior for images (similar to how it works with event messages).  Do you concur?

**extended version**
I don't see a good reason to prevent imagery from being added to the fixed position display.  I could see a user wishing to see the last url of an image in a simple display.

It does seem to be a bug that when you add an image to a layout, there is no value displayed.  I'd expect to see the image url.  Instead I see nothing, and errors are logged in the console:

![screen shot 2017-10-02 at 10 50 42 am](https://user-images.githubusercontent.com/226503/31091442-da657230-a75f-11e7-8e93-ef627016db96.png)
![screen shot 2017-10-02 at 10 51 36 am](https://user-images.githubusercontent.com/226503/31091449-dd2cc1f8-a75f-11e7-8973-2e70fbba6cf8.png)

```
FixedController.js:364 Uncaught TypeError: Cannot read property 'source' of undefined
    at FixedController.updateView (FixedController.js:364)
    at FixedController.js:348
    at TelemetryAPI.js:273
    at Array.forEach (<anonymous>)
    at TelemetryAPI.js:272
    at plugin.js:64
```

**Comment by charlesh88** [comment]
@larkin Agree, and given our current approach to FP objects, displaying the URL makes the most sense.

**Comment by larkin** [comment]
@charlesh88 Okay, updated issue.
